documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,250 @@ You may obtain a copy of the License at
 
-### Stream Segments and ReaderGroups
+### Stream Segments and Reader Groups
 
-Stream Segments are important to understanding the way Reader Groups work.
+Stream Segments play a major role in understanding the way Reader Groups work.
 
 ![Stream Segment](img/segment.readergroup.png) 
 
-Pravega assigns each Reader in a ReaderGroup zero or more Stream Segments to
-read from.  Pravega tries to balance out the number of Stream Segments each
-Reader is assigned.  In the figure above, Reader B1 reads from 2 Stream Segments
-while each of the other Readers in the Reader Group have only 1 Stream Segment
-to read from.  Pravega makes sure that each Stream Segment is read by exactly
-one Reader in any ReaderGroup configured to read from that Stream. As Readers
-are added to the ReaderGroup, or Readers crash and are removed from the
-ReaderGroup, Pravega reassigns Stream Segments so that Stream Segments are
-balanced amongst the Readers.
+Pravega assigns _zero_ or more Stream Segments to each Reader in a Reader Group. Pravega tries to balances the number of Stream Segments assigned to each Reader. In the figure above, **Reader B1** reads from two Stream Segments (**Segment 0** and **Segment 3**), while the other Reader Group (**Reader B2**, **Reader B3**) have only only one Stream Segment to read from. Pravega makes sure that each Stream Segment is read exactly by one Reader in any Reader Group configured with that Stream. Irrespective of  Readers being added to the Reader Group or removed from the Reader Group due to crash, Pravega reassigns Stream Segments to maintain balance amongst the Readers.
 
 The number of Stream Segments in a Stream determines the upper bound of
-parallelism of readers within a ReaderGroup – the more Stream Segments, the more
-separate, parallel sets of Readers we can have consuming the Stream. In the
-above figure, Stream1 has 4 Stream Segments.  That means that the largest
-effective Reader Group would contain 4 Readers.  Reader Group named "B" in the
-above figure is not quite optimal.  If one more Reader was added to the
-ReaderGroup, each Reader would have 1 Stream Segment to process, maximizing read
-parallelism.  However, the number of Readers in the ReaderGroup increases beyond
-4, at least one of the Readers will not be assigned a Stream Segment.
-
-If Stream1 in the figure above experienced a Scale-Down event, reducing the
-number of Stream Segments to 3, then Reader Group B as depicted would have an
+parallelism of readers within a Reader Group. If there are more Stream Segments, different Reader Groups and many parallel sets of Readers can effectively consume the Stream. In the
+above figure, **Stream 1** has four Stream Segments. The largest effective Reader Group would contain four Readers. **Reader Group B** in the above figure is not quite optimal. If one more Reader was added to the Reader Group, each Reader would have one Stream Segment to process, and maximizes the read
```

Comment:
Please, change: `, and maximizes the` -> `, thus maximizing read parallelism.`

Reply:
done